src/main/java/org/jabref/logic/xmp/XMPUtilReader.java

```diff
@@ -30,7 +35,7 @@ private XMPUtilReader() {
      * @param path The path to read the XMPMetadata from.
      * @return The XMPMetadata object found in the file
      */
-    public static Optional<XMPMetadata> readRawXMP(Path path) throws IOException {
+    public static Optional<List<XMPMetadata>> readRawXMP(Path path) throws IOException {
```

Comment:
A `List` should always be non-null and thus it does not makes sense to wrap an `Optional` around it. The not-present case corresponds to an empty list, which you can check using `isEmpty()`.

Reply:
Thanks for this comment. Done 👍